functions/_common/grapherTools.ts

```diff
@@ -304,3 +312,53 @@ export function rewriteMetaTags(
 
     return rewriter.transform(page)
 }
+
+export function getGrapherTableWithRelevantColumns(
```

Comment:
This is a bit of a dangerous change because the grapher endpoint also hits this code path, and we could miss some columns in this manual filtering step that explorers need.

That said, we've talked about making downloaded tables look nicer, so this might be a good spot to handle that in the future.

Reply:
Done!